app/page.tsx

```diff
@@ -10,27 +10,52 @@ import movieData from "../lib/top_100_movies.json";
 
 const Home = () => {
   const [searchText, setSearchText] = React.useState("");
+  const [filteredMovieData, setFilteredMovieData] = React.useState(movieData);
 
   const handleSearch = (e: React.FormEvent<HTMLInputElement>) => {
     e.preventDefault();
-    setSearchText(e.currentTarget.value);
+    const newSearchText = e.currentTarget.value;
+    setSearchText(newSearchText);
+    const newFilteredMovieData = movieData.filter((movie) =>
+      movie.title.includes(newSearchText.toLocaleLowerCase())
```

Comment:
bug: If i look for "dark", I'd expect to get 2 results:
- The Dark Knight
- The Dark Knight Rises

But I get none. Can you find out why?


<details>
<summary>Answer:</summary>

(read only after you tried to look for yourself)

The `newSearchText` is transformed to lowercase but the `movie.title` isn't.
So if I search "dark" but the title name is "Dark Knight", it won't match.

You'll need to add `.toLowerCase()` to the `movie.title` too:
```
movie.title.toLowerCase().includes(newSearchText.toLowerCase())
```

</details>

P.S. You've used `toLocaleLowerCase`, but in this case as the list of movies is English for everyone, `toLowerCase` would probably fit better to ensure consistent names for everyone.

Reply:
I think you missed this comment too.